.github/workflows/pr-checks.yaml

```diff
@@ -0,0 +1,158 @@
+name: "Pull Request Checks"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - edited
+  merge_group:
+    branches:
+      - main
+    types:
+      - checks_requested
+
+jobs:
+  pull-reqeust-checks:
+    needs:
+      - title-validation
+      - associated-issue
+    runs-on: ubuntu-22.04
+    if: ${{ !cancelled() }}
+    steps:
+      - if: contains(needs.*.result, 'failure')
+        run: echo "Failed due to ${{ contains(needs.*.result, 'failure') }}" && exit 1
```

Comment:
```suggestion
```

Reply:
Maybe I am missing something @dmihalcik-virtru but why don't we need this if we want it to be a required status check on a pull request?

Reply:
yeah, this is fine if you don't want to make the `pr` job required which was what i thought you were going with. can you please fix the spelling?